client/components/card/test/__snapshots__/index.js.snap

```diff
@@ -2,720 +2,504 @@
 
```

Comment:
Not directly related to this PR, but I think those tests should care only about HTML code that was shallow rendered.